Plugins/Public/base_plugin/BuildModule.cpp

```diff
@@ -171,48 +185,54 @@ bool BuildModule::Timer(uint time)
 	return false;
 }
 
-void BuildModule::LoadState(INI_Reader &ini)
+void BuildModule::LoadState(INI_Reader& ini)
```

Comment:
No, I specifically made sure not to use any new mandatory fields. Many have in fact been removed and their values are being populated at runtime from config.

Initial tests for reverse compatibility were successful but it's been a few month since I last ran them, there will be another batch of tests coming very soon when Haste has a bit of capacity.